terraform/modules/monitoring/variables.tf

```diff
@@ -33,3 +33,43 @@ variable "public_key_service_name" {
   type        = string
   description = "The Public Key Cloud Run service to monitor."
 }
+
+variable "jvs_ui_service_name" {
+  type        = string
+  description = "The JVS-UI Cloud Run service to monitor."
+}
+
+variable "notification_channel_email" {
+  type        = string
+  description = "The Email address where alert notifications send to."
+}
+
+variable "is_prod_env" {
+  type        = bool
+  description = "If deploying to a prod environment, true if is prod env, otherwise false."
+  default     = false
+}
+
+variable "cert_rotator_500_response_threshold" {
```

Comment:
same 5xx

Reply:
Sorry, forgot to change those 5xx for variable names. Fixed.